Script/material.dat

```diff
@@ -333,7 +333,8 @@ solid
     PriceModifier = 6000;
     NameStem = "sapphire";
     Alpha = 200;
-    AttachedGod = MELLIS;
+    AttachedGod = VALPURUS;
```

Comment:
Oh! It took me quite long to realize we are talking about sapphires! I wanted diamonds to be valpurian and hardening into black diamonds of Mortifer, the sapphire is a error on my part to begin with.

Sapphires are back to Mellis, hardening is fixed to diamond > black diamond, and I'm inattentive and dumb.